Code/BuildSystem/AzurePipelines/Windows-x64.yml

```diff
@@ -90,9 +104,9 @@ jobs:
     inputs:
       cmakeArgs: -DCMAKE_PREFIX_PATH=$(System.DefaultWorkingDirectory)\ThirdPartyWinx64\Qtvs141x64; -DEZ_QT_DIR="$(System.DefaultWorkingDirectory)\ThirdPartyWinx64\Qtvs141x64" -DEZ_ENABLE_FOLDER_UNITY_FILES=0 -DEZ_BUILD_OPENXR=1 -DEZ_BUILD_PHYSX=1 -DEZ_BUILD_FMOD=1 -DEZ_BUILD_GAMES=1 -DEZ_COMPILE_ENGINE_AS_DLL=1 -DEZ_ENABLE_QT_SUPPORT=1 -DCMAKE_SYSTEM_VERSION=10.0.18362.0 -DCMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION=10.0.18362.0 -A "x64" -G "Visual Studio 16 2019" ../
```

Comment:
-DEZ_BUILD_PHYSX=1 -DEZ_BUILD_FMOD=1 -DEZ_BUILD_GAMES=1

They all should be removed.
PhysX will be removed at some point and the default switches to Jolt. One of them will always be on.
Fmod is now on by default anyway.
The GAMES flag doesn't exist anymore.

Reply:
Do I need to add a flag for jolt?

Reply:
Nope, just remove the stuff and once I switch the default I can run a clean on CI to make it switch there as well :)